vic/vic_run/src/surface_fluxes.c

```diff
@@ -250,7 +250,7 @@ surface_fluxes(bool                 overstory,
 
     energy->advection = 0;
     energy->deltaCC = 0;
-    if (snow->swq > 0) {
+
         snow_flux = energy->snow_flux;
     }
     else {
```

Comment:
Yes, I somehow deleted this line accidentally without re-testing the run. My fault, will fix it.